README.md

```diff
@@ -1,56 +1,22 @@
-
-### In Progress
-#### Create basic application structure
-We are setting up JS Code Structure, a bare map viewer with panning and investigate PWA technologies. This includes a set up of CI/CD and deployment processes. To ease collaboration , basic documentation is implemented
-
-#### Basic mapviewer with MVP feature set
-A basic viewer consisting of basic Permalink,/ URL mgmt., basic dataset mgmt., basic infoboxes (incl improvements), basic drawing, basic iFrame support and basic 3D.
-
-### Future
-#### Map window
-We will redefine the user learning journeys and refresh the approach to change language, display coordinates/ show mouse positons, Zoom to rectangle, copyright and possibly fullscreen.
-
-#### Navigation & orientation
-We will redefine the user learning journeys and refresh the approach to Geographical search (centre on coordinates), Geographical search (place name or postcode),Geolocation,
-
-#### 3D
-We will redefine the user learning journeys and refresh the approach to Navigation, Mouse position, Feature tooltip, WMS import: preset, WMS import: permalink, KML import, Time support vector, Time support Raster.
-
-#### Selection and display of data
-We will redefine the user learning journeys and refresh the approach to Full text search for data, Search for data attributes, Topic switcher, Search for data (catalogue/topics), Change the cartographic background, Change the visibility of the selected dataset, Change the order of the selected datasets, Remove selected datasets, View additional information about a data-set. View information about an object on the map , View information about an object on the map including highly customized tooltips with logic and dynamic data fetching) (EXPERTTOOLTIP),  Position information, Aerial photography, Time slider, Time copy/compare.
-
-#### Linking & integration
-We will redefine the user learning journeys and refresh the approach to Send/save/iFrame of the current map, Save/send a link to an object, Use parameters to call up a map (incl. drawings, external ogc layers), Mobile, Mobile Add link to homescreen/App, Mobile Offline SAVE VIEW DELETE.
-
-#### Draw & measure functions
-We will redefine the user learning journeys and refresh the approach to Add symbol, Add text, Add line / polygon, Measure, Measure Azimut, Profile, Share drawing / KML
-
-#### Advanced functions
-We will redefine the user learning journeys and refresh the approach to Import preset, Import URL WMS/WMTS/GeoJSON and KML, import: PRINT, Import permalink, Compare, Keyboard shortcuts, Report problem, Landscape mode (mobile)
-
-#### Print
-We will redefine the user learning journeys and refresh the approach to Print and Printing grids and annotation
-
-#### 3D nextgen
-We will define the user learning journeys to shadow calculation
 
-#### Draw & measure functions nextgen
-We will define the user learning journeys to shadow snapping and routing
+See [ROADMAP.md](ROADMAP.md)
```

Comment:
Why not removing this and add the link to ROADMAP.md directly on top of README in the Table of Content ?